src/components/Create/components/pages/ReviewDeploy/components/ProjectTokenReview/ProjectTokenReview.tsx

```diff
@@ -26,7 +26,7 @@ export const ProjectTokenReview = () => {
   return (
     <div className="flex flex-col gap-y-10 pt-5 pb-12 md:grid md:grid-cols-4">
       <ReviewDescription
-        title={t`Initial mint rate`}
+        title={t`Issuance rate`}
```

Comment:
```suggestion
        title={t`Total issuance rate`}
```

Reply:
Was a bit ambivalent on this one, but you're right. Adding.